infrastructure/util/cookie.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"fmt"
+	"github.com/PTSS-Support/identity-service/domain/errors"
+	"github.com/PTSS-Support/identity-service/infrastructure/constants"
+	"github.com/gin-gonic/gin"
+)
+
+func SetRefreshTokenCookie(ctx *gin.Context, refreshToken string) {
+	ctx.SetCookie(
+		constants.RefreshTokenCookie,
+		refreshToken,
+		constants.RefreshTokenDuration,
+		constants.CookiePathAuth,
+		"",
+		true,
+		true,
```

Comment:
Ik neem aan dat de twee `true`'s voor HttpOnly en Secure zijn. Die moeten ook uit de env gehaald worden. Zie de [`AuthenticationFilter`](https://github.com/PTSS-Support/PTSS-Support-Quarkus-Base-Repo/blob/development/src/main/kotlin/org/ptss/support/security/AuthenticationFilter.kt) van Daan voor een voorbeeld.

Reply:
jup klopt, maar waarom zouden we dit in een env halen? Zou ter alle tijden true moeten zijn lijkt me? Of verwacht je dat het uiteindelijk false moet zijn?

Reply:
In dev is het makkelijker om zo te kunnen switchen van true naar false voor bugfixes en troubleshooting

Reply:
ik wil graag vermijden dat iemand ze op 'false' houdt en deployed naar productie :)
debugging kan je overigens doen secure of niet, http only or not ook gwn. Insomnia works fine namelijk, maar ook gewoon de browser.

Reply:
Exact, en dat probleem krijg je dus ook door het in de constants te zetten in niet in de configmaps. Dit introduceert dus alleen maar problemen. 
Voor debugging, dan kan je ook de token daadwerkelijk aanpassen ipv alleen zien, en mogelijk de path even snel aanpassen voor een localhost url.